bazel/osx/crosstool/CROSSTOOL.tpl

```diff
@@ -1270,8 +1275,7 @@ toolchain {
     config_name: "c++-link-interface-dynamic-library"
     action_name: "c++-link-interface-dynamic-library"
     tool {
-      tool_path: "wrapped_clang"
-      execution_requirement: "requires-darwin"
+      tool_path: "DUMMY_TOOL"
```

Comment:
@zuercher I don't know any of the details here, but FYI 0.5.3 is out of that changes anything.